package.json

```diff
@@ -53,14 +53,14 @@
     "grunt-contrib-copy": "^0.7.0",
     "grunt-contrib-jshint": "^0.11.2",
     "grunt-contrib-uglify": "^0.7.0",
-    "grunt-jsdoc": "^0.5.8",
+    "grunt-jsdoc": "^2.0.0",
```

Comment:
Hi @adon-at-work,

Mainly because of the older version of `grunt-jsdoc` (15 months old), where a newer version also includes security fixes to 2 dependencies which `grunt-jsdoc` relies on: `moment` and `marked` packages.

See more information here: https://www.bithound.io/github/yahoo/xss-filters/master/dependencies/npm/grunt-jsdoc